src/utils.js

```diff
@@ -666,3 +666,18 @@ export function getBidderRequest(bidder, adUnitCode) {
       .filter(bid => bid.bidder === bidder && bid.placementCode === adUnitCode).length > 0;
   }) || { start: null, requestId: null };
 }
+
+/**
+ *
+ * https://stackoverflow.com/a/34890276/428704
+ * @export
+ * @param {array} xs
+ * @param {string} key
+ * @returns {${key_value}: ${groupByArray}, key_value: {groupByArray}}
+ */
+export function groupBy(xs, key) {
```

Comment:
lol, it's not. Sorry... should have been clear: I didn't actually expect a change here. More of a fun FYI.

Many general-purpose utils libraries have this function, including [lodash](https://lodash.com/docs/4.17.4#groupBy), [async](https://caolan.github.io/async/groupBy.js.html), [the scala stdlib](http://www.scala-lang.org/api/2.12.1/scala/collection/immutable/List.html#groupBy[K](f:A=>K):scala.collection.immutable.Map[K,Repr]), etc. I was just pointing out how close this comes to "the bleeding edge" of general-purpose code.